util/settings/settings.go

```diff
@@ -550,12 +550,14 @@ func (mgr *SettingsManager) GetResourceOverrides() (map[string]v1alpha1.Resource
 	}
 
 	crdGK := "apiextensions.k8s.io/CustomResourceDefinition"
+	crdPrsvUnkn := "/spec/preserveUnknownFields"
 
 	switch diffOptions.IgnoreResourceStatusField {
 	case "", "crd":
 		addStatusOverrideToGK(resourceOverrides, crdGK)
```

Comment:
As the generic version of this function is available now, can this function be removed and use generic function instead.
Eg: 
`addIgnoreDiffItemOverrideToGK(resourceOverrides, "*/*", status)    // status = "/status"`